documentation/projects/proposals/dark_mode/20240313-project_proposal_dark_mode.md

```diff
@@ -0,0 +1,96 @@
+# 2024-03-13 Project Proposal
+
+**Author**: @fcoveram
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat
+- [ ] @zackkrida
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+Include a feature that changes the UI from light to a dark color scheme.
+
+### Motivation
+
+Dark mode is a customization level that is in the spirit of adapting the UI to
+user's contexts. The benefits span a11y, personal preferences set on the
+devices, energy efficiency in devices, and an alternative browsing experience of
+visual content due to the background contrast.
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+Refine Search Experience
+
+## Requirements
+
+<!-- Detailed descriptions of the features required for the project. Include user stories if you feel they'd be helpful, but focus on describing a specification for how the feature would work with an eye towards edge cases. -->
+
+1. Add the dark theme to the site and set the Light theme as default.
```

Comment:
Because the current theme is light and the unexpected change is not a good way of introducing the feature. With that in mind, we should include an onboarding process.